<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainTab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0" colspan="3">
         <widget class="QTableView" name="taskTable">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <widget class="QLabel" name="loadAverage">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Load average: x.xx x.xx x.xx</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="uptime">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>Uptime: xx:xx:xx</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tasks">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Tasks: xxxx total, xxxx running, xxxx sleeping, xxxx stopped, xxxx zombie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>159</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="searchModeLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <italic>true</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text">
           <string>Search Mode: Enter [ESC] to exit search mode</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_1">
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mem</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QProgressBar" name="memBar">
            <property name="font">
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Swp</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="swpBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>CPU</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QProgressBar" name="cpuBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="text">
             <string>&amp;Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="niceButton">
            <property name="font">
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>&amp;Nice</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="killButton">
            <property name="font">
             <font>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>&amp;Kill</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="text">
             <string>&amp;Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="performanceTab">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-198</y>
             <width>893</width>
             <height>853</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>CPU</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QChartView" name="cpuChartView">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>200</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>Mem</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="memDetailLabel">
                 <property name="text">
                  <string>(MiB) xxxxx.x total, xxxxx.x free, xxxxx.x shared, xxxxx.x buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QChartView" name="memChartView">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>200</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>Swap</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="swapDetailLabel">
                 <property name="text">
                  <string>(MiB) xxxxx.x total, xxxxx.x free</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QChartView" name="swapChartView">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>200</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="systemTab">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Operation System</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="osInfoLabel">
             <property name="text">
              <string>Manjaro</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>5</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>CPU</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="cpuInfoLabel">
             <property name="text">
              <string>Intel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cpuCacheInfoLabel">
             <property name="text">
              <string>Cache xxxx KB</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cpuCoreInfoLabel">
             <property name="text">
              <string>x cores, x threads </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>5</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Hostname / Work Group</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Hostname: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="hostnameLabel">
             <property name="text">
              <string>cyx-pc</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>User: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="userLabel">
             <property name="text">
              <string>cyx</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Group:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="groupLabel">
             <property name="text">
              <string>cyx</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>5</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>DateTime</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Current:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="currentDatetimeLabel">
             <property name="text">
              <string>xxx xxx xx xx:xx:xx 20xx</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_13">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Last login:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lastLoginDatetimeLabel">
             <property name="text">
              <string>xxx xxx xx xx:xx:xx 20xx</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="shutdownButton">
            <property name="minimumSize">
             <size>
              <width>116</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Shutdown</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rebootButton">
            <property name="minimumSize">
             <size>
              <width>116</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Reboot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
